### PR TITLE
My first Pull Request

### PR DESCRIPTION
**Pull request by omer-creator**
##  Microverse first project : Linters

  Add:
- [x]  index.html
- [x]  styles.css
- [x]  .gitignore(ignore node_modules and test.md)
- [x]  .hintrc
- [x]  .stylelintrc.json
- [x]  linters.yml
- [x] README.md

### CODE REVIEW

**Review by OyePriscilla — CHANGES_REQUESTED** [review]
Hi @omer-creator  😃  , 

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!

so-close-so-close-seth-meyers

## Highlights
Github flow observed✔️
Commit messages meaningful ✔️ 

## Required Changes ♻️

- [x] Kindly ensure to use this [Readme](https://github.com/microverseinc/readme-template) file template, and ensure to enter all the necessary details; this will give your repo a professional touch and guide anyone that wants to access your project.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 1]
```diff
@@ -1,2 +1,200 @@
...
+
+b/)
+
+
+
+
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
```
Comment (Qoosim) [inline README.md:150]: - [ ] Kindly remove these lines from your README. They are not needed here.

File: test.md [hunk 2]
```diff
@@ -0,0 +1 @@
+<p> This file should be ignored by git </p>
```
Comment (Qoosim) [inline test.md:1]: - [ ] I can see that you've tried to add this to your `.gitignore` file but still appears on your repo. Kindly follow this [guide](https://www.educative.io/answers/how-to-add-a-file-to-gitignore-on-github) to rectify that issue.
test-file

**Review by Qoosim — CHANGES_REQUESTED** [review]
#  *Status: Approved* :tada: :heavy_check_mark: 

Hi @omer-creator , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
-  No linter errors. :heavy_check_mark: 
-  Professional REAMDE file. :heavy_check_mark: 
-  Correct GitHub Flow. :heavy_check_mark: 



## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**